corehq/motech/openmrs/tasks.py

```diff
@@ -188,13 +189,9 @@ def import_patients_to_domain(domain_name, force=False):
     :param domain_name: The name of the domain
     :param force: Import regardless of the configured import frequency / today's date
     """
-    today = datetime.today()
     for importer in get_openmrs_importers_by_domain(domain_name):
-        if not force and importer.import_frequency == IMPORT_FREQUENCY_WEEKLY and today.weekday() != 1:
-            continue  # Import on Tuesdays
-        if not force and importer.import_frequency == IMPORT_FREQUENCY_MONTHLY and today.day != 1:
-            continue  # Import on the first of the month
-        # TODO: ^^^ Make those configurable
```

Comment:
configurability no longer a TODO :trollface:

Reply:
lol! In the two years since this code was written, nobody has asked for the day of the week or day of the month to be configurable, so I figured that makes it "YAGNI".